src/utils/mapUtils.ts

```diff
@@ -309,7 +308,8 @@ export function numToString(num: number, p?: number): string {
 }
 
 export function formatNumber(x, formatChar = ' ', roundPrecision?: number): string {
-    if (isNumber(x)) {
+    /** !!!!! not to be merged before TEST. DONT let me do it in a PR !!!!!!!!!!! */
```

Comment:
So shall we merge in this PR ? :stuck_out_tongue_closed_eyes:

Reply:
refixed, ouf